expath-pkg.xml.tmpl

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<package xmlns="http://expath.org/ns/pkg" name="http://exist-db.org/apps/monex" abbrev="monex" version="0.9.5"
+<package xmlns="http://expath.org/ns/pkg" name="http://exist-db.org/apps/monex" abbrev="@project.name@" version="@project.version@"
```

Comment:
That is what this PR did. This PR keeps `expath-pkg.xml.tmpl` and generates `expath-pkg.xml`, and the generated `expath-pkg.xml` file is listed in `.gitignore` so changes to it are not committed back to source.